python/lib/dcoscli/tests/integrations/test_node.py

```diff
@@ -93,9 +94,8 @@ def test_node_log_missing_slave():
         ['dcos', 'node', 'log', '--mesos-id=bogus'])
 
     assert returncode == 1
-    assert stdout == b''
     stderr_str = str(stderr)
-    assert 'No files exist. Exiting.' in stderr_str
+    assert 'HTTP 404' in stderr_str
```

Comment:
The error message looked better before, can we have a custom one instead of `HTTP 404`?